libraries/Microsoft.Bot.Builder.AI.QnA/Utils/CardHelper.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class CardHelper
```

Comment:
Don't name public classes "Helper"

Reply:
Consider more meaningful things that point at the responsibility of the class. If it builds cards, then it oculd be named something like CardBuilder, QnACardBuilder, etc